Classes/Cache/ExposedValueObjectCache.php

```diff
@@ -0,0 +1,35 @@
+<?php
+declare(strict_types=1);
+
+namespace Netlogix\JsonApiOrg\AnnotationGenerics\Cache;
+
+use Neos\Flow\Annotations as Flow;
+use Netlogix\JsonApiOrg\AnnotationGenerics\Domain\Model\ExposedValueObjectInterface;
+
+/**
+ * @Flow\Proxy(false)
+ */
+class ExposedValueObjectCache
+{
+
+    protected static $entries = [];
+
+    public static function set(string $entryIdentifier, ExposedValueObjectInterface $data)
+    {
+        $className = get_class($data);
+        if (!array_key_exists($className, self::$entries)) {
+            self::$entries[$className] = [];
+        }
+        self::$entries[$className][$entryIdentifier] = $data;
+    }
+
+    public static function get(string $entryIdentifier, string $className): ?ExposedValueObjectInterface
+    {
+        return self::$entries[$className][$entryIdentifier] ?? false;
```

Comment:
This will raise a type error if `$entryIdentifier` is not set.

```suggestion
        return self::$entries[$className][$entryIdentifier] ?? null;
```